TSPL.docc/LanguageGuide/ControlFlow.md

```diff
@@ -1828,6 +1829,136 @@ without wrapping it in an `else` block,
 and it lets you keep the code that handles a violated requirement
 next to the requirement.
 
+## Deferred Actions
+
+Unlike control-flow constructs like `if` and `while`,
+which let you control whether part of your code is executed
+or how many times it gets executed,
+`defer` controls *when* a piece of code is executed.
+You use a `defer` block to write code that will be executed later,
+when your program reaches the end of the current scope.
+For example:
+
+```swift
+var score = 1
+if score < 10 {
+    defer {
+        print(score)
+    }
+    score += 5
+}
+// Prints "6"
+```
+
+<!--
+  - test: `defer-with-if`
+
+  ```swifttest
+  -> var score = 3
+  -> if score < 10 {
+  ->     defer {
+  ->         print(score)
+  ->     }
+  ->     score += 5
+  -> }
+  <- 8
```

Comment:
This test block looks like it's gotten out of sync with the listing that will render.

Reply:
Fixed in a6235080